src/grammar.coffee

```diff
@@ -396,6 +396,7 @@ grammar =
   # A "meta-property" access e.g. `new.target`
   MetaProperty: [
     o 'NEW_TARGET . Property',                  -> new MetaProperty LOC(1)(new IdentifierLiteral $1), LOC(3)(new Access $3)
+    o 'IMPORT_META . Property',                 -> new MetaProperty LOC(1)(new IdentifierLiteral $1), LOC(3)(new Access $3)
```

Comment:
Re your question in the issue thread re `LOC()`, it is used to make sure that newly created node class objects get correct location data assigned to them:

By default, the grammar will assign the location data spanned by *all* of the left-hand-side tokens to the "top-level" node returned by the grammar rule. So eg here, the `new MetaProperty()` will be assigned the correct location data corresponding to eg `import.meta` in the source code

But for "inner" node class objects created by grammar rules, they won't get correct location data assigned to them by default. So eg here in order for the `new IdentifierLiteral()` to get assigned the location data corresponding to `import` in the source code, we use `LOC(1)(new IdentifierLiteral ...)` to say "assign the location data of the *first* token of this grammar rule's left-hand-side (ie `IMPORT_META`) to this `new IdentifierLiteral()`"

You can see the implementation of `LOC()` in the `o()` function at the top of `src/grammar.coffee`, which does a little magic to allow the grammar rules to output Jison-compatible rules